scripts/run_frontend_tests.py

```diff
@@ -108,6 +119,35 @@ def run_dtslint_type_tests() -> None:
         sys.exit('The dtslint (type tests) failed.')
 
 
+def get_file_spec(file_path: str) -> str | None:
+    """Returns the spec file path for a given file path.
+
+    Args:
+        file_path: str. The path of the file.
+
+    Returns:
+        str | None. The path of the spec file if it exists, otherwise None.
```

Comment:
Note that `None` is also returned if the input file isn't even a JS/TS file

Reply:
Done.